### PR TITLE
[chore] Try tuning some renovate bot things

### LINKED ISSUE #29784: renovatebot should not run CI checks on initial PR
### Component(s)

_No response_

### Describe the issue you're reporting

The initial PR created by renovatebot should not trigger CI, as it will always fail. Instead, it should skip checks, until the opentelemetrybot commit to run tidy on the PR.

### PR DESCRIPTION
**Pull request by TylerHelmuth**
Closes https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/29784

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -7,9 +7,6 @@
     "constraints": {
       "go": "1.20"
     },
-    "extends": [
-      "config:base"
-    ],
```
Comment (TylerHelmuth) [inline renovate.json:12]: I cant find a definition for `base` anywhere.  I don't think this is doing anything, but I'd like to be proven wrong.

**Review by songy23 — APPROVED** [review]
LGTM as long as opentelemetrybot is guaranteed to run after renovate bot (which I believe is the case)

File: renovate.json (hunk 1 above)
Comment (codeboten) [reply renovate.json:12]: Huh... it was definitely in the config example i found, but i wonder if we should change this to another preset https://docs.renovatebot.com/presets-config/#configrecommended

Comment (codeboten) [reply renovate.json:12]: hah, it looks like it `base` -> `recommended` https://github.com/renovatebot/renovate/blob/5ff18174495251d2e3e204be770547776a14fa9c/lib/config/presets/common.ts#L17

Comment (TylerHelmuth) [reply renovate.json:12]: added it back as `config:recommended`

**Review by codeboten — APPROVED** [review]

**Pull request merged**